### PR TITLE
Add Spanish (Spain) translation

### PR DESCRIPTION
**Pull request by droberin**
Add Spanish (Spain) translation

### CODE REVIEW

File: locale/es.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ es:
   'Timeout: 60s': ~
   Mouse: Ratón
   Keyboard: Teclado
-  Message Alert: ~
+  Message Alert: Alertar mensaje
   NodeDB Reset\nFactory Reset: Borrar NodeDB\nReini. Fábrica
```
Comment (mverch67) [inline locale/es.yml:33]: need double quotes

"NodeDB Reset\nFactory Reset": "Borrar NodeDB\nReini. Fábrica"

File: locale/es.yml [hunk 2]
```diff
@@ -46,21 +46,21 @@ es:
...
-  Signal Scanner: Escáner señal
+  Signal Scanner: Escáner de señal
   Trace Route: Trazar ruta
   Neighbors: Vecinos
   Statistics: Estadísticas
-  Packet Log: Reg. Paquetes
+  Packet Log: Registro de paquetes
   Settings & Tools: Config y herramientas
   Node Search: Buscar nodo
   "Light\nDark": "Claro\nOscuro"
```
Comment (mverch67) [inline locale/es.yml:56]: This line can be removed

File: locale/es.yml [hunk 3]
```diff
@@ -14,13 +14,13 @@ es:
...
   Settings: Configuración
   Settings (advanced): Configur. avanzada
-  Locations Map: ~
+  Locations Map: Mapa de localizaciones
   no chats: sin chats
-  Node Options: ~
+  Node Options: Opciones de Nodos
   Short Name: Nombre Corto
   Long Name: Nombre Largo
   <unset>: ~
```
Comment (mverch67) [inline locale/es.yml:26]: No Spanish word for unset (or unused, empty) ?

Comment (droberin) [reply locale/es.yml:26]: _(un)set_ has like 30 or 40 meanings. An accurate translation could be _sin_ + (infinite verb) _establecer, declarar, definir_, etc.
but _empty_ could work better as _vacío_. and has the same visual length as _unset_. 
gonna try that

**Pull request merged**